release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,1534 @@
+logs from fluentd to external fluentd (that can then ingest into ES). `ES_COPY`
+is removed from the installer and the documentation.
+
+When `openshift_installer` is run for logging to upgrade to 3.7, the installer
+now checks for `ES_COPY` in the inventory and fails the upgrade with:
+
+----
+msg: The ES_COPY feature is no longer supported. Please remove the variable from your inventory
+----
+
+See
+xref:../install_config/aggregate_logging.adoc#fluentd-log-external-elasticsearch[Aggregating
+Container Logs] for more information.
+
+[[ocp-37-expose-es-as-a-route]]
+==== Expose Elasticsearch as a Route
+
+By default, Elasticsearch (ES) deployed with OpenShift aggregated logging is not
+accessible from outside the logging cluster. This enables a route for external
+access to ES for those tools that want to access its data.
+
+You now have direct access to ES using only your OpenShift token and have the
+ability to provide the external ES and ES Ops hostnames when creating the server
+certificate (similar to Kibana). Ansible tasks now simplify route deployment.
+
+[[ocp-37-removed-metrics-and-logging-deployers]]
+==== Removed Metrics and Logging Deployers
+
+The metrics and logging deployers bare now replaced with `playbook2image` for
+`oc cluster up` so that `openshift-ansible` is used to install logging and
+metrics:
+
+----
+$ oc cluster up --logging --metrics
+----
+
+Check metrics and pod status:
+
+----
+$ oc get pod -n openshift-infra
+$ oc get pod -n logging
+----
+
+[[ocp-37-prometheus]]
+==== Prometheus (Technology Preview)
+
+{product-title} operators deploy Prometheus (currently in
+xref:ocp-37-technology-preview[Technology Preview] on a {product-title} cluster,
+collect Kubernetes and infrastructure metrics, and get alerts. Operators can see
+and query metrics and alerts on the Prometheus web dashboard, or bring their own
+Grafana and hook it up to Prometheus.
+
+See xref:../nstall_config/cluster_metrics.adoc#openshift-prometheus[Prometheus
+on OpenShift] for more information.
+
+[[ocp-37-integrated-approach-to-adding-hosa]]
+==== Integrated Approach to Adding Hawkular OpenShift Agent (Techology Preview)
+
+Hawkular OpenShift Agent (HOSA) remains in
+xref:ocp-37-technology-preview[Technology Preview]. It is packaged and can now
+be installed with the `openshift_metrics_install_hawkular_agent` option in the
+installer by setting it to `true`.
+
+See
+xref:../install_config/cluster_metrics.adoc#metrics-ansible-variable[Enabling
+Cluster Metrics] for more information.
+
+[[ocp-37-developer-experience]]
+=== Developer Experience
+
+[[ocp-37-template-instantation-api]]
+==== Template Instantiation API
+
+Clients can now easily invoke a server API instead of relying on client logic.
+
+* To set parameters, create a secret with values.
+* Create a `TemplateInstance` containing the whole template you want to
+instantiate, and a reference to the secret created above.
```

Comment:
this references "the secret created above" but there are no instructions for creating the secret.  Anyway we have official docs for this, can we just point there instead of duplicating it here?
https://docs.openshift.org/latest/rest_api/examples.html#template-instantiation